Sources/Brave/Frontend/Sync/SyncSettingsTableViewController.swift

```diff
@@ -307,11 +322,15 @@ class SyncSettingsTableViewController: SyncViewController, UITableViewDelegate,
 
   @objc
   private func onSyncInternalsTapped() {
-    let syncInternalsController = syncAPI.createSyncInternalsController().then {
-      $0.title = Strings.braveSyncInternalsTitle
+    askForAuthentication() { [weak self] status, error in
+      guard let self = self, status else { return }
+      
+      let syncInternalsController = syncAPI.createSyncInternalsController().then {
```

Comment:
Do you need `self.syncAPI`?

Reply:
Yeah great catch fix at https://github.com/brave/brave-ios/pull/7878/commits/2873a2ad927225f02157498543c1e57e183f24ee